tests/views/test_user.py

```diff
@@ -74,6 +80,9 @@ def test_user_me_with_logged_in_user(self):
         )
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.csrf_cookie_set, True)
+        self.assertEqual(response.cookies.get('edx_csrf_token').key, 'edx_csrf_token')
```

Comment:
This assert looks funny, maybe we can replace it with assertIsNotNone ?
```suggestion
        self.assertIsNotNone(response.cookies.get('edx_csrf_token'))
```

Reply:
Yup right !